src/System.Linq.Expressions/tests/SequenceTests/SequenceTests.cs

```diff
@@ -3661,29 +3530,15 @@ public static void ConvertUnsignedToSigned()
         [Fact]
         public static void ConvertCheckedSignedToUnsigned()
         {
-            try
-            {
-                Func<ulong> f = Expression.Lambda<Func<ulong>>(Expression.ConvertChecked(Expression.Constant((sbyte)-1), typeof(ulong))).Compile();
-                ulong result = f();
-                Assert.False(true);
-            }
-            catch (OverflowException)
-            {
-            }
+            Func<ulong> f = Expression.Lambda<Func<ulong>>(Expression.ConvertChecked(Expression.Constant((sbyte)-1), typeof(ulong))).Compile();
+            Assert.Throws<OverflowException>(() => f());
         }
 
         [Fact]
         public static void ConvertCheckedUnsignedToSigned()
         {
-            try
-            {
-                Func<sbyte> f2 = Expression.Lambda<Func<sbyte>>(Expression.ConvertChecked(Expression.Constant(UInt64.MaxValue), typeof(sbyte))).Compile();
-                sbyte result = f2();
-                Assert.False(true);
-            }
-            catch (OverflowException)
-            {
-            }
+            Func<sbyte> f2 = Expression.Lambda<Func<sbyte>>(Expression.ConvertChecked(Expression.Constant(UInt64.MaxValue), typeof(sbyte))).Compile();
```

Comment:
Nit: this could just be ```f``` instead of ```f2```.  Not sure why the original was ```f2```, but then again there's a bunch of things about the original tests that don't make sense :)